lib/widgets/game_view.dart

```diff
@@ -1,11 +1,20 @@
 import 'package:flutter/material.dart';
+import 'package:tic_tac_toe/widgets/rules_buttons.dart';
 
 import '../grids_page.dart';
 import '../model/boards.dart';
 import '../model/game.dart';
 import '../styles/colors.dart';
 import 'board.dart';
 
+var winner = '';
+
+late final Game game1;
+int inRowValue = 3;
```

Comment:
These two variables should not be global variables, but part of state of the widget RulesChangingButtons.

Reply:
we dont need it if we are not checking the board size. removed

Reply:
I still see them in PR.

Reply:
Deleted

Reply:
I still see global variables in PR:

```
int inRowValue = 3;
```

All them should be moved to state classes of widgets, if they are needed.

Reply:
Moved it to the widget